indico/modules/events/contributions/client/js/index.jsx

```diff
@@ -8,13 +8,48 @@
 /* global showUndoWarning:false, setupListGenerator:false, setupSearchBox:false */
 /* global reloadManagementAttachmentInfoColumn:false */
 
+import fileTypesURL from 'indico-url:event_editing.api_file_types';
+
 import React from 'react';
 import ReactDOM from 'react-dom';
 
 import 'indico/modules/events/util/types_dialog';
+import EditableSubmissionButton from 'indico/modules/events/editing/components/EditableSubmissionButton';
 import {PublicationSwitch} from 'indico/react/components';
+import {camelizeKeys} from 'indico/utils/case';
+import {indicoAxios, handleAxiosError} from 'indico/utils/axios';
 
 (function(global) {
+  global.setupEditableSubmissionButton = async function setupEditableSubmissionButton() {
+    const editableSubmissionButton = document.querySelector('#editable-submission-button');
```

Comment:
While testing I noticed that this element may not exist (e.g. no submission rights) and that will mean you'll get `undefined`, which will blow up the whole thing.

Reply:
Is it the best approach to use `if` inside the setup function or not call the setup function at all in that case?

Reply:
I don't think it makes much sense to call `setupEditableSubmissionButton` if you don't have the button